src/index.css

```diff
@@ -1,8 +1,7 @@
 
 .title{text-align: center;}
-
 .regForm{width:100%;}
-.inputbox{margin:20px;}
-.inputL{margin:50px;}
-.inputC{margin:0 20px;}
+.inputCheck{margin:20px;}
+.inputLabel{margin-left:50px;display:inline-block;width:180px;}
```

Comment:
write as following will be clear:
```
.inputLabel {
  margin-left:50px;
  display:inline-block;
  width:180px;}
```

Reply:
I userd to do this. but it will be hard to find when style become complex